AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -18,6 +18,42 @@
 #import "ASThread.h"
 
 /**
+ @abstract Object to hold UITextView's pending UITextInputTraits
+**/
+@interface _ASTextInputTraitsPendingState : NSObject
```

Comment:
Although it would be nice to develop a generalized way to handle property bridging, this is fine for now.  cc @Adlai-Holler as you authored the ASPendingStateController.  cc @rahul-malik as you are the local code generation guru :)